src/core/EdFi.DataManagementService.Core/Response/FailureResponseWithErrors.cs

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Core.External.Model;
+
+namespace EdFi.DataManagementService.Core.Response;
+
+/// <summary>
+/// Failure response with error lists
+/// </summary>
+internal record FailureResponseWithErrors(
+    /// <summary>
+    /// A human-readable explanation specific to this occurrence of the problem.
+    /// </summary>
+    string detail,
+    /// <summary>
+    /// A URI reference [RFC3986] that identifies the problem type. This specification encourages that, when
+    /// dereferenced, it provide human-readable documentation for the problem type
+    /// (e.g., using HTML [W3C.REC-html5-20141028]).  When this member is not present, its value is assumed to be
+    /// "about:blank".
+    /// </summary>
+    string type,
+    /// <summary>
+    /// A short, human-readable summary of the problem type.It SHOULD NOT change from occurrence to occurrence
+    /// of the problem, except for purposes of localization(e.g., using proactive content negotiation;
+    /// see[RFC7231], Section 3.4).
+    /// </summary>
+    string title,
+    /// <summary>
+    /// The HTTP status code([RFC7231], Section 6) generated by the origin server for this occurrence of the problem.
+    /// </summary>
+    int status,
+    /// <summary>
+    /// Gets or sets a correlation id that is logged with the details of the exception.
+    /// </summary>
+    string? correlationId,
+    /// <summary>
+    /// Contains arrays of validation errors keyed by JSON Path (intended for use by a domain expert).
+    /// </summary>
+    Dictionary<string, string[]>? validationErrors,
```

Comment:
I don't know if `validationErrors` and `errors` are part of the response anymore as of ODS/API 7.2, but they are not in the responses for invalid references.